proofs/tree.go

```diff
@@ -41,28 +41,33 @@ import (
 	"strconv"
 	"strings"
 
+	"regexp"
+
+	"github.com/centrifuge/go-merkle"
 	"github.com/centrifuge/precise-proofs/proofs/proto"
-	"github.com/go-bongo/go-dotaccess"
 	"github.com/golang/protobuf/descriptor"
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/timestamp"
 	"github.com/iancoleman/strcase"
-	"github.com/centrifuge/go-merkle"
 )
 
+const DefaultSaltsLengthSuffix = "Length"
+
 type TreeOptions struct {
```

Comment:
Please add a description to this. I'd copy the comments from the README to the code here so it shows up in the godocs.